boring-sys/Cargo.toml

```diff
@@ -18,6 +18,7 @@ include = [
     "/*.toml",
     "/LICENSE-MIT",
     "/cmake/*.cmake",
+    "/deps/boringssl/src/util/32-bit-toolchain.cmake",
```

Comment:
Can we move this below L22 and add the fips version of it as well? Not sure if it's even relevant for fips but might as well

Reply:
Well, I forgot about the fips version, actually the fips version also has this cmake configuration file, I have added it now